code/modules/events/antag_ghost_respawn.dm

```diff
@@ -1,3 +1,12 @@
+/datum/random_event/major/antag
+	///Can this event be given a custom spawn location
+	var/targetable = FALSE
+	var/turf/custom_spawn_turf = null
+	admin_call(var/source)
+		if (src.targetable)
+			var/custom_loc = alert("Custom spawn location?","[src.name]","Default","Custom")
```

Comment:
```suggestion
			var/custom_loc = alert("Custom spawn location?","[src.name]","Default","Custom") == "Custom"
```
Otherwise Default will still prompt for picker I think. Or is there some voodoo with that string?

Reply:
`User was undev'ed for this PR`